core/network/besu/contracts/interop/asset_locks.proto.sol

```diff
@@ -906,6 +906,293 @@ library AssetExchangeAgreementCodec {
 
 }
 
+struct HybridAssetExchangeAgreement {
```

Comment:
Don't commit this file, can you delete this and add it in gitignore?
This is generated after building protos right?